examples/AspNetCoreMvc/Properties/launchSettings.json

```diff
@@ -55,7 +55,13 @@
         "OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED": "true",
         "OTEL_DOTNET_AUTO_TRACES_PLUGINS": "Examples.AspNetCoreMvc.OtelSdkPlugin, Examples.AspNetCoreMvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
         "OTEL_SERVICE_NAME": "StartupHook.Self-hosted",
-        "OTEL_TRACES_EXPORTER": "otlp"
+        "OTEL_TRACES_EXPORTER": "otlp",
```

Comment:
This is the default and could be removed.

Reply:
Nice call out, removed `OTEL_LOGS_EXPORTER` from the profile. Will create a new PR to remove the `OTEL_TRACES_EXPORTER` and other default values from profile.

Reply:
#1135